azure-pipelines.yml

```diff
@@ -96,27 +96,7 @@ jobs:
   - bash: ci/scripts/rust-format.sh $SYSTEM_PULLREQUEST_TARGETBRANCH
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: rustfmt
-  - bash: |
-      set -e
-      # Azure-specific installation:
-      # The latest version of shellcheck available to ubuntu-18.04 is 0.4.6-1.
-      # We need a newer version that provides the --warning flag.
-      SC_VERSION="v0.8.0"
-      SHELLCHECK_FILE="shellcheck-$SC_VERSION.linux.x86_64.tar.xz"
-      SHELLCHECK_URL="https://github.com/koalaman/shellcheck/releases/download/$SC_VERSION/$SHELLCHECK_FILE"
-      SC_SHA256=ab6ee1b178f014d1b86d1e24da20d1139656c8b0ed34d2867fbb834dad02bf0a
-      echo "Installing Shellcheck $SC_VERSION"
-      curl -L -o "$SHELLCHECK_FILE" "$SHELLCHECK_URL" --silent --show-error
-      echo "${SC_SHA256}  ${SHELLCHECK_FILE}" | sha256sum -c
-      tar -xJf $SHELLCHECK_FILE
-      sudo cp "shellcheck-$SC_VERSION/shellcheck" /usr/bin
-      # Run shellcheck
-      echo "Checking shell scripts..."
-      ci/scripts/run-shellcheck.sh || {
-        echo -n "##vso[task.logissue type=error]"
-        echo "Shellcheck failed. Run ci/scripts/run-shellcheck.sh to see errors."
-        exit 1
-      }
+  - bash: ci/bazelisk.sh test //quality:shellcheck_check
```

Comment:
@miles Note that I got rid of the repro instructions. I figured it's easy enough to figure out how to repro failures now that this is a one-liner (the command should appear in the CI log). I'm happy to bring back the error message if you feel strongly.

Reply:
I would lean towards adding it back just because it seems likely that someone less familiar with bazel/CI might get hung up on this check and end up wasting time backtracking.